simulator/index.md

```diff
@@ -28,26 +28,52 @@ You will also need Python installed.
 
 ### Installing the simulation
 
-
-1. [Download the simulation](https://github.com/srobo/competition-simulator/releases/download/sr2023.1/competition-simulator-sr2023.1.zip), and unzip it somewhere on your computer.
-2. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+1. Create a directory, perhaps called `simulation` where you will store your robot code.
+2. [Download the simulation](https://github.com/srobo/competition-simulator/releases/download/sr2023.1/competition-simulator-sr2023.1.zip)
+  and unzip it as a subdirectory of that directory:
+  ```
+  simulation
+  ├── competition-simulator-<version>
+  │   ├── ...
+  │   └─ worlds
+  │       └── Arena.wbt
+  └── robot.py
+  ```
+  If there is not an existing `robot.py` an example one will be created when the simulator first runs.
+3. Using the Webots IDE, open the `worlds/Arena.wbt` file.
 
 You may receive a warning about your computer's GPU not being good enough, which can be ignored.
 
+<div class="info">
+  On recent versions of macOS you may need to give Webots permission to access the directory where you have extracted the simulation files.
+</div>
 
 #### Changing your version of Python
 
-If webots is picking up the incorrect version of Python, you'll need to change it.
-This can be done using **Tools** &rarr; **Preferences** &rarr; **General** &rarr; **Python command** (or **Webots** &rarr; **Preferences** <kbd>⌘</kbd><kbd>,</kbd> on a Mac).
+If webots is not picking any version of Python or is picking up the wrong one then you'll need to change it.
+When this happens Webots will print errors to its console and your robot will not move.
+
+You will need the full path to the version of Python that you want to use.
+This will vary based on the system you have.
+One way to find the path is by launching the version of Python that you want to
```

Comment:
Are we comfortable that all students will know how to launch python?

Reply:
Not entirely. I'm hoping that in most cases it'll be "double click the icon on the desktop"?